docs/src/inputs/dropdownDeprecated/dropdownDeprecated.jsx

```diff
@@ -1275,565 +1212,706 @@ class ModulesDropdown extends React.Component {
+                        onChange={this._onSelectionCustomOptionsChange.bind(this)}
+                        options={selectionCustomOptions}
+                        placeholder="Select a user"
+                        selection
+                        selectionUnderline
+                        value={this.state.selectionCustomOptionsValue}
+                    />
+
+                    <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
+                        {selectionUnderlineSample}
+                    </Highlighter>
+
+                    {/* Theme */}
+                    <Header anchor="theme" size="large" style={{ marginTop: '55px' }} sub>
+                        Theme
+                        <Header.Subheader>
+                            Changes the menu's color.
+                        </Header.Subheader>
+                    </Header>
 
-                        <Dropdown
-                            onChange={this._onSelectionMultipleCreatableChange.bind(this)}
-                            options={selectionMultipleOptions}
-                            placeholder="Select some tags"
-                            selection
-                            selectionCreatable
-                            selectionMultiple
-                            value={this.state.selectionMultipleCreatableValue}
-                        />
+                    <Dropdown button placeholder="Button Dropdown" theme="dark">
+                        <Dropdown.Item label="Option 1" />
+                        <Dropdown.Item label="Option 2" />
+                        <Dropdown.Item label="Option 3" />
+                        <Dropdown.Item label="Option 4" />
+                    </Dropdown>
 
-                        <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                            {selectionCreatableSample}
-                        </Highlighter>
+                    <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
+                        {themeSample}
+                    </Highlighter>
+                </div>
+            );
+        }
 
-                        {/* Selection Mobile */}
-                        <Header anchor="selection-mobile" size="large" style={{ marginTop: '55px' }} sub>
-                            Selection Mobile
-                            <Header.Subheader>
-                                A Dropdown Selection can be user friendly on mobile devices. Go ahead, shrink that browser.
-                            </Header.Subheader>
-                        </Header>
+        return (
+            <Main page="headers">
+                <Main.Content>
+                    <Header size="large" style={{ marginTop: '55px' }}>
+                        The prop 'button' has been deprecated.
+                        <br />
+                        Please use <Link to={{ pathname: '/molecules/dropdown-button' }}>Dropdown Button</Link> component instead.
```

Comment:
This link is probably not correct anymore

Reply:
Fixed.

Reply:
Resolving.